tests/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(test_srcs test_bacio.pf)
+
+add_pfunit_ctest (bacio_test
+  TEST_SOURCES test_bacio_mod.pf
+  LINK_LIBRARIES bacio_4_static
```

Comment:
I think this will fail if we don't build static library.
A solution is to add a block before the `add_pfunit_ctest()` like such:
```
# If both static and shared libraries are built,
# use the 1st one to link the tests
list( GET LIB_TARGETS 0 _LIB_TARGET )
```
And then on this line, replace `bacio_4_static` with `${_LIB_TARGET} `

Reply:
Fixed

Reply:
How about now?

Reply:
yes. This is great.